schemas/telemetry/debug-schema.json

```diff
@@ -9,19 +9,60 @@
       "$ref": "_common-schema.json"
     },
     {
-      "required": [
-        "status",
-        "message"
-      ],
+      "required": ["telemetry"],
       "properties": {
-        "status": {
-          "type": "string",
-          "description": "Level/severity of the log",
-          "const": "debug"
-        },
-        "message": {
-          "type": "string",
-          "description": "Body of the log"
+        "telemetry": {
+          "type": "object",
+          "description": "The telemetry information as it should be indexed",
+          "required": ["status", "message"],
```

Comment:
are we sure to want to make both of these required ? maybe at some point we will want to add a metric telemetry and not send any log message for example ?

Reply:
this is just for a debug telemetry event. A metric telemetry event will be defined in another file, and indeed the status/message won't be mandatory

Reply:
oh, makes sense. lgtm then!